docs/conf.py

```diff
@@ -120,6 +120,14 @@
 
 # -- Options for HTML output -------------------------------------------
 
+# Set canonical URL from the Read the Docs Domain
+html_baseurl = os.environ.get("READTHEDOCS_CANONICAL_URL", "")
+
+html_context = {}
+# Tell Jinja2 templates the build is running on Read the Docs
+if os.environ.get("READTHEDOCS", "") == "True":
+    html_context["READTHEDOCS"] = True
```

Comment:
Looking at [the RTD blogpost]( https://about.readthedocs.com/blog/2024/07/addons-by-default/ ), guess we need this

```suggestion
# Tell Jinja2 templates the build is running on Read the Docs
if os.environ.get("READTHEDOCS", "") == "True":
    if "html_context" not in globals():
        html_context = {}
    html_context["READTHEDOCS"] = True
```

Sorry didn't have a good grasp on what was happening there (and honestly still don't). Guessing there is some RTD magic happening under-the-hood